operator/pkg/util/util_test.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import (
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestListFileWithSuffix(t *testing.T) {
```

Comment:
Could you please show me the execution log of this test? I can't see it from [the unit test workflow log](https://github.com/karmada-io/karmada/actions/runs/6505927839/job/17670796208?pr=4130).

Reply:
That is because we didn't enable unit tests for the operator. Just sent #4140 for it.